earth_enterprise/src/fusion/autoingest/plugins.src/MercatorRasterProject.src

```diff
@@ -1025,6 +1025,8 @@ uint32 MercatorRasterProjectAssetVersionImplD::BuildIndex(
     packgenkids.push_back(packgen->GetRef());
     packgenassets.push_back(packgen->GetAssetRef());
     packgenvers.push_back(packgen);
```

Comment:
This is part of the reason things aren't getting written out. We're saving handles to all the packgens in this vector, so none of them will be written to disk or removed from the cache. That said, there's one packgen per image (I think) and multiple packlevels, and we should be writing those out to disk, so there should still be a big improvement.